src/plugins/isup.lua

```diff
@@ -0,0 +1,36 @@
+url = require("socket.url")
+https = require("ssl.https")
+
+
+local plugin = {}
+
+local messages = { "The webserver hasn't sent a final response, status %s",
+                   "The website is up status %s",
+                   "The website is up with a %s redirect",
+                   "Client error, webserver returned status %s",
+                   "Server returning error %s" }
+
+plugin.help = 'Usage: isup <website>'
+
+plugin.main = function(args)
+    local website = args.message
+    if args.modules.config.debug then
+        print(website)
+    end
+    if not website then
+        args.modules.irc.privmsg(args.target, 'Give me a website or hostname to check')
```

Comment:
probably, we want to `return` at this point so we don't keep going through the rest (which would likely explode at line 23 (on `website:find…`)